### PR TITLE
Document how to use with pkg

### PR DESCRIPTION
**Pull request by thgreasi**
Resolves: #973
Change-type: patch
Signed-off-by: Thodoris Greasidis <thodoris@balena.io>

---
##### Contributor checklist

- [ ] Includes tests
- [ ] Includes typings
- [ ] Includes updated documentation
- [ ] Includes updated build output

### CODE REVIEW

**Comment by thgreasi** [comment]
@balena-ci rebase

File: README.md [hunk 1]
```diff
@@ -43,6 +43,26 @@ In the browser the balena SDK doesn't use the following dependencies:
 
 For the future we're looking at ways to automatically exclude these in downstream bundles. See [#254](https://github.com/balena-io/balena-sdk/issues/254) for more information.
 
+### Bundling with pkg
+
+Because the balena SDK includes builds for various ecmascript versions and the way that it loads the appropriate versions using [@balena/es-version](https://github.com/balena-io-modules/balena-es-version)
+packagers like [pkg](https://github.com/vercel/pkg) are not able to automatically detect which assets should be included in the build output. In such case you need to manually define a `pkg` section in
+your package.json, which will instruct `pkg` to bundle all the required assets:
```
Comment (pdcastro) [inline README.md:50]: A refactoring of the sentences for clarity:
```
The balena SDK includes builds for various ECMAScript versions that are
dynamically selected at runtime (using
[@balena/es-version](https://github.com/balena-io-modules/balena-es-version)).
For this reason, packagers like [pkg](https://github.com/vercel/pkg) are not
able to automatically detect which assets to include in the output package. The
following sample `pkg` section should be added to your application's
`package.json` file to instruct `pkg` to bundle the required assets:
```

File: README.md [hunk 2]
```diff
@@ -43,6 +43,26 @@ In the browser the balena SDK doesn't use the following dependencies:
...
+      "node_modules/balena-sdk/**/*.js"
+    ],
+    "assets": [
+      "node_modules/balena-pine/**/*",
+      "node_modules/pinejs-client-core/**/*"
+    ]
+  }
+```
+
+For more information, please refer to the respective (documentation from the `pkg` project)[https://github.com/vercel/pkg#config].
```
Comment (pdcastro) [inline README.md:64]: Looks like the square brackets and the parentheses were swapped (markdown link syntax).

**Comment by thgreasi** [comment]
Thanks for the awesome suggestions @pdcastro :+1:

**Review by pdcastro — APPROVED** [review]

**Pull request merged**